KiCAD 5.1.4 compatibility fix (py2&3)

easyw opened the pull request.

easyw committed:
k 5.14 compatibility (py2&3)

Fix compatibilty issues with:
KiCAD 5.1.4
Py2
Py3
Added Net Combo box
Added button icon image

easyw committed:
improved button icon

easyw committed:
update name

Shackmeister commented:
not sure if this is part of the compatibility fixes or a seperate issues. but running the script, lowering via copper size to fx 0.5mm. crashes the plugin with the following message:
![Skærmbillede fra 2019-09-27 00-38-36](https://user-images.githubusercontent.com/5279723/65729675-6e92ca80-e0bf-11e9-9d1c-85d270c5e5b9.png)

easyw committed:
removing useless debug message

easyw committed:
adding try:except on processing pads to avoid crash

easyw committed:
adding a temporary workaround for Kicad Nigthly

adding a temporary workaround for Kicad Nigthly (pcbnew.py have removed DrawingsList() )

easyw committed:
adding butto to toolbar

Virtex6 commented:
Patch doesn't work with my 5.99.0-165 (git hash g2e3d781bf). The call of area.HitTestInsideZone() fails as this function is removed in my version. It runs if i set the return value to false, but the keepout areas are filled with vias.

 #https://github.com/easyw/kicad-action-scripts/commit/ac41373ffc70a733810d7637c93a9ab360e5de8d  As far as I know self.pcb.DrawingsList() return the internal list m_drawings. In my version the same return value is given by self.pcb.Drawings()

easyw committed:
minor fix for kicad nightly

To be fixed: nightly is missing 'HitTestInsideZone' function for checking keepout zones inside zones

easyw commented:
hi @Virtex6 @jsreynaud 
I've patched pcb.DrawingsList() w/ pcb.Drawings()
and temporarly also HitTestInsideZone() but this will not work correctly with keepout zones... we would need to check an alternative option to HitTestInsideZone()
PS @jsreynaud  thanks for merging 😄
